perception/autoware_compare_map_segmentation/src/voxel_grid_map_loader/voxel_grid_map_loader.hpp

```diff
@@ -93,13 +92,15 @@ class VoxelGridMapLoader
   bool debug_ = false;
 
 public:
-  typedef VoxelGridEx<pcl::PointXYZ> VoxelGridPointXYZ;
-  typedef typename pcl::Filter<pcl::PointXYZ>::PointCloud PointCloud;
-  typedef typename PointCloud::Ptr PointCloudPtr;
+  using VoxelGridPointXYZ = VoxelGridEx<pcl::PointXYZ>;
+  using PointCloud = typename pcl::Filter<pcl::PointXYZ>::PointCloud;
```

Comment:
@badai-nguyen 
(This is unrelated to this PR but appeared as a change so I will leave a message)
This seems to be a typo.

Reply:
Thank you, I will fix it.
Where should I change it?

Reply:
@yhisaki 
For this one, it is unrelated to your changes (typedef to using), but rather why we are defining
`pcl::Filter<pcl::PointXYZ>::PointCloud` as `PointCloud`. Maybe it is just semantics but I mentioned Dai-san just to be sure

Reply:
@knzo25 @yhisaki I'm sorry, I was my typo. I will fix it right after this PR merged.

Reply:
> Maybe it is just semantics but I mentioned Dai-san just to be sure

@knzo25 the reason is that I was following defination at https://pointclouds.org/documentation/filters_2include_2pcl_2filters_2voxel__grid_8h_source.html (line 217 ).
Anyway, I agree that it's better to rename it and make a change at https://github.com/autowarefoundation/autoware.universe/pull/9181
Could I as your review on that?